shell/platform/darwin/ios/ios_surface.h

```diff
@@ -33,6 +33,10 @@ class IOSSurface {
 
   virtual std::unique_ptr<Surface> CreateGPUSurface() = 0;
 
+  // Creates a secondary surface using the supplied GrContext, if supported.
+  // Otherwise, the same as CreateGPUSurface.
+  virtual std::unique_ptr<Surface> CreateGPUSurface(GrContext* gr_context) = 0;
```

Comment:
Can we just remove the no argument `CreateGPUSurface` and make the argument nullable?

Reply:
Done